schema.sql

```diff
@@ -7,4 +7,21 @@ CREATE TABLE animals (
     weight_kg DECIMAL(5, 2));
 
 ALTER TABLE animals
-    ADD COLUMN species VARCHAR(255);
\ No newline at end of file
+    ADD COLUMN species VARCHAR(255);
+
+
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INTEGER
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+ALTER TABLE animals
+    DROP COLUMN species,
+    ADD COLUMN species_id INTEGER,
+    ADD COLUMN owner_id INTEGER;
```

Comment:
- [ ] @axcell-0 Great work altering the animal's table, however, you forgot to state they are foreign keys eg  ` ALTER TABLE animals ADD species_id int REFERENCES species(id);`
![image](https://github.com/axcell-0/vet-clinics/assets/61408860/80502938-e895-492c-ac70-7380bb290761)